test/test.go

```diff
@@ -584,7 +584,7 @@ func (h *Helper) Cleanup() {
 
 // ReadManifest returns the manifest in the current directory.
 func (h *Helper) ReadManifest() string {
-	m := filepath.Join(h.pwd(), "manifest.json")
+	m := filepath.Join(h.pwd(), "manifest.toml")
```

Comment:
let's create a local constant in this test package too, and use that. it's not as good as one constant, but it's still better than string literals all over.